maven-scm-providers/maven-scm-provider-hg/src/test/java/org/apache/maven/scm/provider/hg/command/blame/HgBlameCommandTckTest.java

```diff
@@ -26,6 +26,8 @@
 
 import java.util.List;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
I don't understand why this import is added; it is the only change in this file?

Reply:
In junit3 you inherited this method (w/ same signature from TestCase), but in Junit4 you need to use hamcrest, or Assert or alike, hence static import but no other change (as method sig is same).